src/CLI/actioner/sonic-cli-neighbors.py

```diff
@@ -211,8 +211,12 @@ def run(func, args):
         elif 'sonic-neighbor:output' in response.keys():
             status = response['sonic-neighbor:output']
             status = status['response']
-            if (status != "Success"):
+            if "255" in status:
+                print "Unable to clear all entries, please try again"
```

Comment:
As discussed, it is expected for Linux to return error when there are lot of ARP entries..please try atleast 3 times and send error from backend.